recipes/chartify/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "chartify" %}
+{% set version = "2.3.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3f991749eb85c90bb527eb08f242799325ad0f891f3744c9b2ed653fa54ece50
+
+build:
+  number: 0
+  skip: True  # [py2k]
```

Comment:
Could this be a noarch package?

Reply:
The build failed when I tried to build it as a noarch package.
https://circleci.com/gh/conda-forge/staged-recipes/31783?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
You need to have `python >=3.5 ` in addition to `noarch` so that it will skip python 2.7